plugin/route53/README.md

```diff
@@ -29,10 +30,13 @@ route53 [ZONE:HOSTED_ZONE_ID...] {
    to external hosts (eg. used to resolve CNAMEs). If no **ADDRESS** is given, CoreDNS will resolve
    against itself. **ADDRESS** can be an IP, an IP:port or a path to a file structured like
    resolv.conf (**NB**: Currently a bug (#2099) is preventing the use of self-resolver).
+* `credentials_file` used for reading the credential file and setting the profile name for a given zone.
```

Comment:
make it just `credentials`?

Reply:
`credentials` by itself didn't seem intuitive to me and felt it could be easily confused with `aws_access_key`, but I'm ok with changing it.